### PR TITLE
Add integration with Travis CI

### LINKED ISSUE #15: Add integration with Travis CI
Please configure integration with TravisCI:
* [x] [Enable repo testing in TravisCI](https://travis-ci.com/kpi-web-guild/django-girls-blog-slsh1o)
* [x] Add TravisCI badge (master branch) to `README.md` with a single commit. (click Build Status next to octocat icon on [this page](https://travis-ci.com/kpi-web-guild/django-girls-blog-slsh1o))
* [x] Create new branch called `features/15-travis-ci-integration` and do all the following steps within that branch.
* [x] Add `.travis.yml` using `pre-commit` as a script. Set it up to run Python 3.8 env and add Pip caching. Ensure you install `pre-commit` before running tests:
```yaml
language: python
python: 3.8
cache: pip
before_install: pip install -U pre-commit
script: pre-commit run --all-files
```
* [x] Push the branch and create a Pull Request for it.

### PR DESCRIPTION
**Pull request by slsh1o**
All seems to be ok.
Except:

> Add .travis.yml using pre-commit as a script. Set it up to run Python 3.8 env and add Pip caching. Ensure you install pre-commit before running tests: 

Here I just copy-pasted your code. Looks like it is what I actually should do but if something else let me know please.

@serhii73 let me know if it's ok that I add you to all my PRs. If it's to much I won't add you as frequently as I did it :D

Also traditionally ask for review @dark-lord671 and @ZVitaliy

Close #15

### CODE REVIEW

**Comment by serhii73** [comment]
> @serhii73 let me know if it's ok that I add you to all my PRs. If it's to much I won't add you as frequently as I did it :D

All good :+1:

**Comment by serhii73** [comment]
Excuse me @webknjaz 
Why Travis?
Why not GitHub actions?

**Comment by webknjaz** [comment]
@serhii73 there's a lot of options, we cannot try all of them at once. Maybe later.

**Comment by webknjaz** [comment]
@dark-lord671 changed the username

**Review by webknjaz — CHANGES_REQUESTED** [review]
Plz exclude commits that belong to the `features/pre-commit-integration` branch by rebasing on top of master.

Commit message improvements:

> Add Travis CI badge

*where to?*

Also:
```diff
-Add Travic CI config file
+Add Travis CI config file
```

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: python
+python: 3.8
+cache: pip
+before_install: pip install -U pre-commit
+script: pre-commit run --all-files
```
Comment (webknjaz) [inline .travis.yml:5]: This'll fail once we start adding actual py files. But let's deal with it in a separate PR.

**Comment by slsh1o** [comment]
> Plz exclude commits that belong to the `features/pre-commit-integration` branch by rebasing on top of master.

If I will fix commits messages from #17 and than merge `features/pre-commit-integration` w/ master branch will it enough?

> Commit message improvements:
> 
> > Add Travis CI badge
> 
> _where to?_

I thought it's obvious where changes happens. Or if there are several commits in a row it would be better to add also explicit description in which file I add something?

_My text sounds a little rude but I really just want to know how to do it better😅_

> Also:
> 
> ```diff
> -Add Travic CI config file
> +Add Travis CI config file
> ```

Can't find where this happened😅

**Comment by webknjaz** [comment]
> If I will fix commits messages from #17 and than merge `features/pre-commit-integration` w/ master branch will it enough?
> 

You should reply after the quotes not before, preserving the natural order of messages.

In the case of merging #17, I'll still demand a rebase here.

> Can't find where this happened
> 

commit-typo

> I thought it's obvious where changes happens. Or if there are several commits in a row it would be better to add also explicit description in which file I add something?

I'm not sure which quote this refers to but whenever you make a commit, it should be atomic and the reader should understand what it's about w/o looking into other places. You should maintain a good Git history for other humans and your future self, otherwise, it's hard to track the decisions and thought flows that lead to making them back to the past.

Imagine you see a piece of `git log` that looks like this:
```console
* fix
|
* Add Travis CI badge
|
* Fix Travis CI badge
|
* Add Travis CI badge
```
Can you say, w/o checking the contents of each commit, that the Travis CI badge was added to docs and to the README? What was fixed? Where?

**Comment by slsh1o** [comment]
>You should reply after the quotes not before, preserving the natural order of messages.

Oh sorry, edited my message.

>Can you say, w/o checking the contents of each commit, that the Travis CI badge was added to docs and to the README? What was fixed? Where?

Now I got what you mean. Thx.

**Comment by webknjaz** [comment]
P.S. It shouldn't necessarily be a full file name but it's good if it's a distinguishable pointer to some concrete place.

**Comment by slsh1o** [comment]
Rebase this branch on top of master and make message of 4d9ca724 more clear.

**Comment by webknjaz** [comment]
Could as well fix this:

> Add Travic CI config file

**Review by webknjaz — APPROVED** [review]

**Pull request merged**